linkerd/app/outbound/src/http/logical/policy/router.rs

```diff
@@ -145,6 +145,7 @@ where
             policy::BackendDispatcher::Forward(addr, ref metadata) => mk_concrete(
                 concrete::Dispatch::Forward(Remote(ServerAddr(addr)), metadata.clone()),
             ),
+            policy::BackendDispatcher::Null => mk_concrete(concrete::Dispatch::Null),
```

Comment:
Do you think it's worth mentioning that building a concrete target is required for the stack to work correctly but that these requests will fail before? Ah, might be a lot of information to just put in a comment 🤷🏻